src/agora/txrx/txrx_argos.cc

```diff
@@ -22,6 +22,83 @@ void PacketTXRX::LoopTxRxArgos(size_t tid) {
       std::min((radio_lo + radios_per_thread), cfg_->NumRadios()) - 1;
 
   threads_started_.fetch_add(1);
+
+  long long time0 = 0;
+  if (cfg_->HwFramer() == false) {
```

Comment:
Since this is a big chunk of code, we should add some comments about what this chunk of code does (maybe we should also explain what HwFramer()==false means here since it is not very intuitive).

Reply:
Added some comments here. See if it makes sense.